src/ripple/app/ledger/impl/LedgerCleaner.cpp

```diff
@@ -301,7 +311,8 @@ class LedgerCleanerImp
 
         if(! getApp().getLedgerMaster().fixIndex(ledgerIndex, ledgerHash))
         {
-            m_journal.debug << "ledger " << ledgerIndex << " had wrong entry in history";
+            m_journal.debug << "ledger " << ledgerIndex
+                            << " had wrong entry in history";
```

Comment:
@HowardHinnant Explain why `<<` at the beginning of a line is better than `<<` at the end of the previous line